android/julius/src/main/java/bvschaik/julius/JuliusMainActivity.java

```diff
@@ -0,0 +1,120 @@
+package bvschaik.julius;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.os.Handler;
+import android.os.Looper;
+import android.widget.Toast;
+import org.libsdl.app.SDLActivity;
+
+public class JuliusMainActivity extends SDLActivity
```

Comment:
Please keep to the Java code style conventions of putting the curly brace on the same line